### ISSUE TITLE
edge['curved'] not found in edge.py unless g.es["curved"] explicitly set in code.

### ISSUE THREAD

**Issue by jeffreycwitt** [igraph/python-igraph#504]
**Describe the bug**

I'm not sure if this is a bug or not, but I thought it might be an unnoticed breaking change somewhere.

I was following [this tutorial](https://towardsdatascience.com/newbies-guide-to-python-igraph-4e51689c35b4). The code as described in the tutorial seemed to work for the author. But when I ran it with Python 3.10 and python-graph 0.9.9, I got the following error: 

```
~/.pyenv/versions/3.10.0/lib/python3.10/site-packages/igraph/drawing/edge.py in get_label_position(self, edge, src_vertex, dest_vertex)
    157 
    158         # Determine the midpoint
--> 159         if edge['curved']:
    160             (x1, y1), (x2, y2) = src_vertex.position, dest_vertex.position
    161             aux1, aux2 = get_bezier_control_points_for_curved_edge(x1, y1, x2, y2, edge['curved'])

KeyError: 'Attribute does not exist'

```

See my stack overflow post: 
https://stackoverflow.com/questions/70854916/key-error-attribute-does-not-exist-when-trying-to-plot-graph-with-pythong-igraph?noredirect=1#comment125261604_70854916

It was only resolved by explicitly adding:

```
g.es["curved"] = False
```

**To reproduce**
Here's the code that produced the error
```
from igraph import *

# Create graph
g = Graph(directed=True)

# Add 5 vertices
g.add_vertices(5)

# Add ids and labels to vertices
for i in range(len(g.vs)):
    g.vs[i]["id"]= i
    g.vs[i]["label"]= str(i)

# Add edges
g.add_edges([(0,2),(0,1),(0,3),(1,2),(1,3),(2,4),(3,4)])

# Add weights and edge labels
weights = [8,6,3,5,6,4,9]
g.es['weight'] = weights
g.es['label'] = weights

visual_style = {}

out_name = "graph.png"

# Set bbox and margin
visual_style["bbox"] = (400,400)
visual_style["margin"] = 27

# Set vertex colours
visual_style["vertex_color"] = 'white'

# Set vertex size
visual_style["vertex_size"] = 45
```

Again adding `g.es["curved"] = False` resolved the error.

**Version information**
Which version of `python-igraph` are you using and where did you obtain it?

igraph `0.9.9` obtained via `pip install igraph` running python `3.10`

**Comment by szhorvat** [comment]
Possibly related: #474

**Comment by ntamas** [comment]
Or maybe: #501

**Comment by iosonofabio** [comment]
Can you please check the develop branch? Pretty sure this is already fixed

**Comment by iosonofabio** [comment]
Confirm this already works in the `develop` branch, which will be released as `0.10`. I'm closing it here because backporting matplotlib fixes is not really worth it since there are a ton of changes since `0.9.9`.
